packages/bodl/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@bigcommerce/bodl",
```

Comment:
I'm starting to think we should move this into `core/` for now as we'd have to publish this if we wanted to use it. I think we'd rather wait to publish this until we know it is 100% what we want to do. What do you think?

Reply:
I see your point and it makes sense. Another alternative is to call package `catalyst-bodl` so it won't be so generic for now.

Reply:
@chanceaclark moved, please check